src/screens/DiscoveryScreenV2/index.js

```diff
@@ -26,6 +25,12 @@ import NewsFragment from './fragment/NewsFragment';
 import TopicFragment from './fragment/TopicFragment';
 import UsersFragment from './fragment/UsersFragment';
 
+const DiscoveryContainer = ({children}) => {
+  return (
+    <SafeAreaView style={{flex: 1, backgroundColor: COLORS.almostBlack}}>{children}</SafeAreaView>
+  );
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The introduction of `DiscoveryContainer` is a good practice as it abstracts the common layout styling into a separate component, improving readability and reusability. However, ensure that the `COLORS.almostBlack` color is consistent with the rest of your application's theme.
```javascript
const DiscoveryContainer = ({children}) => {
  return (
    <SafeAreaView style={{flex: 1, backgroundColor: COLORS.almostBlack}}>{children}</SafeAreaView>
  );
};
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->